packages/integrations/vercel/src/serverless/adapter.ts

```diff
@@ -245,6 +245,8 @@ You can set functionPerRoute: false to prevent surpassing the limit.`
 				const filesToInclude = includeFiles?.map((file) => new URL(file, _config.root)) || [];
 				filesToInclude.push(...extraFilesToInclude);
 
+				getRuntime(true);
```

Comment:
I mean we should have this part as `validateRuntime()` instead, and `getRuntime()` doesn't have any parameters. The `version` and `major` variables would be duplicated, but I think that's fine.